esmvalcore/config/config-logging.yml

```diff
@@ -10,24 +10,42 @@ formatters:
     format: '%(levelname)-7s [%(process)d] %(message)s'
   debug:
     format: '%(asctime)s UTC [%(process)d] %(levelname)-7s %(name)s:%(lineno)s %(message)s'
+filters:
+  only_cmor:  # only events from CMOR check and generic fixes
+    (): esmvalcore.config._logging.FilterMultipleNames
+    names: [esmvalcore.cmor.check, esmvalcore.cmor._fixes.fix.genericfix]
+    mode: allow
+  no_cmor:  # no events from CMOR check and generic fixes
+    (): esmvalcore.config._logging.FilterMultipleNames
+    names: [esmvalcore.cmor.check, esmvalcore.cmor._fixes.fix.genericfix]
+    mode: disallow
 handlers:
   console:
     class: logging.StreamHandler
     level: INFO
     formatter: console
     stream: ext://sys.stdout
+    filters: [no_cmor]
   simple_log_file:
     class: logging.FileHandler
     level: INFO
     formatter: brief
     filename: main_log.txt
     mode: w
+    filters: [no_cmor]
   debug_log_file:
     class: logging.FileHandler
     level: DEBUG
     formatter: debug
     filename: main_log_debug.txt
     mode: w
+  cmor_log:  # only contains output from CMOR check and generic fixes
+    class: logging.FileHandler
+    level: DEBUG
```

Comment:
maybe INFO or WARNING as default? See me comment in the PR about this

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/pull/2361/commits/1b5974a1cd50ce9099a2a879d0f151c897548fdf